vendor/parse-wyc/lib/parser.mly

```diff
@@ -2352,7 +2338,17 @@ simple_expr:
   | BEGIN ext_attributes seq_expr END
       { Pexp_beginend $3, $2 }
   | BEGIN ext_attributes END
-      { Pexp_construct (mkloc (Lident "()") (make_loc $sloc), None), $2 }
+      { let ext, attrs = $2 in
```

Comment:
This PR would be a good opportunity to preserve begin end in this case.

Reply:
Done